src/fcgen/fastcat_types.yaml

```diff
@@ -492,6 +492,11 @@ commands:
       type: bool
       comment: unused
 
+  - name: fts_check_sensor_safety
```

Comment:
I prefer a name like `enable_fts_guard_fault`

Reply:
It appears our convention is to start with device type so `fts_enable_guard_fault` is my rec instead.